torch/csrc/profiler/kineto_client_interface.cpp

```diff
@@ -71,46 +71,24 @@ class LibKinetoClient : public libkineto::ClientInterface {
 
 } // namespace profiler::impl
 
+extern "C" TORCH_API void global_kineto_init() {
 #if ENABLE_GLOBAL_OBSERVER
-namespace {
-
-int get_init_delay() {
-  const char* delay_c = std::getenv("KINETO_DAEMON_INIT_DELAY_S");
-  if (!delay_c) {
-    return -1;
-  }
-  std::string delay_s{delay_c};
-  try {
-    return std::stoi(delay_s);
-  } catch (const std::invalid_argument& _) {
-    return -1;
+  if (std::getenv("KINETO_USE_DAEMON") != nullptr) {
```

Comment:
Use thread-safe c10::utils::get_env

Reply:
OK, just updated the code, could you please approve the workflows to check whether this update is ok?